pyinterprod/pronto/signature.py

```diff
@@ -115,6 +115,7 @@ def _compare_signatures(matches_file: str, src: Queue, dst: Queue):
                                 0,  # number of collocations
                                 0,  # number of proteins with overlaps
                                 0,  # number of overlapping residues
+                                0, # number of overlapping reviewed residues
```

Comment:
PEP-8 recommends to use at least two spaces before an inline comment:

```suggestion
                                0,  # number of overlapping reviewed residues
```